.github/workflows/create_js_branch.yaml

```diff
@@ -13,20 +13,30 @@ jobs:
     runs-on: ubuntu-latest
     env:
       BRANCH: releases/${{ github.event.inputs.version}}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
+      - name: setup git config
+        run: |
+          git config user.name "GitHub Action"
+          git config user.email "github-action@users.noreply.github.com"
+
       - name: guarantee branch doesn't exist
         run: |
-          if git ls-remote --heads origin ${BRANCH}) ; then 
+          if git ls-remote --exit-code --heads origin ${BRANCH} ; then 
             exit 1
           else
             exit 0
           fi
 
+      - name: install ncc
+        run: |
```

Comment:
We shouldn't install ncc ourselves. If it's not in the package.json it's a problem with whatever repo we're using it in. The next run step of installing dependencies should accomplish this.

Reply:
I added it since I was getting this error.
![image](https://github.com/OliverMKing/action-release-workflows/assets/123775811/d4710f4c-7adb-4dde-8b1b-a9bfdd2399fe)

Reply:
Removed install ncc! 😅